content/applications/inventory_and_mrp/barcode/setup/device_troubleshooting.rst

```diff
@@ -0,0 +1,214 @@
+==============================
+Barcode device troubleshooting
+==============================
+
+Odoo *Barcode* supports three main types of barcode scanners: USB scanners, bluetooth scanners, and
+mobile computer scanners. While configuring each type of scanner, common issues may arise, in which
+the scanners do not work as intended, and Odoo returns errors to the device.
+
+Read the sections below to identify general and unique device issues, related to specific, popular
+types of scanners.
+
+.. seealso::
+   - :doc:`../setup/hardware`
+   - :doc:`../setup/software`
+
```

Comment:
Suggestion to move this seealso:: to the bottom of the doc because the previous sentence "Read the sections below..." made me think these seealso docs were the sections to read 🙂

Reply:
That's a good point! Moving it there to avoid confusion.

Suggestion implemented ✅